mobile-app/app/screens/AppNavigator/screens/Portfolio/components/BalanceText.tsx

```diff
@@ -23,3 +23,12 @@ export function BalanceText ({ symbol, value, ...otherProps }: BalanceTextProps
     </ThemedText>
   )
 }
+export function BalanceTextV2 ({ symbol, value, ...otherProps }: BalanceTextProps & ThemedProps & TextProps): JSX.Element {
```

Comment:
filename is `BalanceText` are you trying to overwrite current code or trying to create a new file?

Reply:
New component

Reply:
Let me transfer the component into a new file